.github/workflows/windows-vcpkg.yml

```
@@ -0,0 +1,36 @@
+name: Windows vcpkg Workflow
+
+on:
+  push:
```

```suggestion
  push:
    branches:
      - master
```
?

Just so we avoid building everything twice when we push a branch directly to the repo to create a PR.

---

It make sense.